softioc/pvlog.py

```diff
@@ -0,0 +1,9 @@
+# Support for PV put logging
+#
+# Importing this module configures PV put logging
+
+import os
+from . import imports
+
+imports.install_pv_logging(
+    os.path.join(os.path.dirname(__file__), 'access.acf'))
```

Comment:
Oh goodness, sorry about all that.  Here it is.

It didn't appear as a rename in the earlier part of the commit.